components/fctl/membership-swagger.yaml

```diff
@@ -197,15 +204,9 @@ paths:
         schema:
           type: string
         required: true
-      - name: deleted
```

Comment:
Doing this case a breaking change for fctl

Reply:
same for `--disabled` 


1/ Do I mark them deprecated ?
2/ Do I keep those query paramaters membership side ? (Map it on --all)